.github/workflows/inbox.yaml

```
@@ -25,10 +25,12 @@ jobs:
       - name: Setup Docker
         uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)